concept/thing/Relation.java

```diff
@@ -114,15 +114,23 @@ default boolean isRelation() {
         @CheckReturnValue
         Map<RoleType.Remote, List<Thing.Remote<?, ?>>> playersMap();
 
+        /**
+         * Retrieves a list of every Thing involved in the Relation.
+         *
+         * @return a list of every Thing involved in the Relation.
+         */
+        @CheckReturnValue
+        Stream<Thing.Remote<?, ?>> players();
+
         /**
          * Retrieves a list of every Thing involved in the Relation, filtered by Role played.
          *
          * @param roles used to filter the returned instances only to ones that play any of the role types.
-         *              If blank, returns all role players.
-         * @return a list of every Thing involved in the Relation.
+         *
+         * @return a list of every Thing involved in the Relation, filtered by Role played.
          */
         @CheckReturnValue
-        Stream<Thing.Remote<?, ?>> players(RoleType... roles);
+        Stream<Thing.Remote<?, ?>> players(List<RoleType> roles);
```

Comment:
Let's refer to `RoleType` as "Role Type" in docs, and `roleType` in method arguments. Let's do this consistently everywhere.